ecosystem/node-checker/src/evaluators/build_evaluators.rs

```diff
@@ -93,6 +97,16 @@ pub fn build_evaluators(
         None => log_did_not_build(&name),
     }
 
+    let name = LatencyEvaluator::get_name();
```

Comment:
nit: we could consider exposing a function on each `Evaluator` that will add the evaluator to the given list. This way we don't need to leak this information here, and we could write a simple loop.